### ISSUE TITLE
ipfs-cluster-service gives wrong error when no configuration initialized

### ISSUE THREAD

**Issue by VictorBjelkholm** [ipfs/ipfs-cluster#373]
With no configuration setup and running `ipfs-cluster-service` gave me the following error:

```
$ ipfs-cluster-service
22:01:17.842 ERROR    service: could not obtain execution lock.  If no other process
is running, remove ~/path/to/lock, or make sure that the config folder is
writable for the user running ipfs-cluster.  Run with -d for more information
about the error lock.go:37
error acquiring execution lock: could not obtain execution lock
```

Following with the suggestion, I added `-d` to get more information and got:

```
$ ipfs-cluster-service -d
22:01:26.710  INFO    service: Initializing. For verbose output run with "-l debug". Please wait... main.go:442
22:01:26.710 DEBUG    service: checking lock lock.go:28
22:01:26.710 DEBUG    service: cannot acquire lock: Lock Create of /home/user/.ipfs-cluster/cluster.lock failed: open /home/user/.ipfs-cluster/cluster.lock: no such file or directory lock.go:31
22:01:26.710 ERROR    service: could not obtain execution lock.  If no other process
is running, remove ~/path/to/lock, or make sure that the config folder is
writable for the user running ipfs-cluster.  Run with -d for more information
about the error lock.go:37
error acquiring execution lock: could not obtain execution lock
22:01:26.710 DEBUG    service: locking not initialized, unlock is noop lock.go:48
```

- [ ] `~/path/to/lock` from the first error message is incorrect, seems to be a placeholder
- [ ] The suggested reason for the error is incorrect, as `~/.ipfs-cluster` does not exists at all, but ipfs-cluster-service doesn't check if the directory actually exists before checking if there is a `cluster.lock` file in there.

**Comment by hsanjuan** [comment]
This has just been fixed by @lanzafame last week :)

**Comment by lanzafame** [comment]
@VictorBjelkholm @hsanjuan everyone happy for this to be closed?

**Comment by VictorBjelkholm** [comment]
Yup, seems to be much better now:

```
$ ipfs-cluster-service daemon
01:24:48.335  INFO    service: Initializing. For verbose output run with "-l debug". Please wait... app.go:485
01:24:48.335 ERROR    service: ipfs-cluster-service config hasn't been initialized.
Please run ipfs-cluster-service init lock.go:29
error acquiring execution lock: stat /home/user/.ipfs-cluster: no such file or directory
```

Thanks @lanzafame

**Comment by hsanjuan** [comment]
Fixed in #370
